readme.md

```diff
@@ -155,6 +155,11 @@ Number of pending promises.
 
 Whether the queue is currently paused.
 
+## Events
+
```

Comment:
Needs to document that `PQueue` is an EventEmitter subclass and that it has inherited certain methods and link to Node.js docs.